docs/src/pages/components/data-grid/selection/selection.md

```diff
@@ -27,7 +27,7 @@ Row selection can be performed with a simple mouse click, or using the [keyboard
 ### Single row selection
 
 Single row selection is enable by default with the `DataGrid` component.
-To unselect a row, hold the <kbd class="key">CTRL</kbd> or <kbd class="key">Shift</kbd> (use <kbd class="key">⌘ Command</kbd> on macOS) key and click on it.
+To unselect a row, hold the <kbd class="key">CTRL</kbd> key and click on it.
```

Comment:
We just have CTRL here. It's a bit confusing 😆

Reply:
In https://github.com/mui-org/material-ui-x/pull/1813#discussion_r645833808, I have tried to argue why Shift is harmful to this feature.

Reply:
Pressing <kbd>Shift</kbd> while clicking the rows selects the text too. It's better to not support this key.

![hNBz68IIRK](https://user-images.githubusercontent.com/42154031/121016339-659e3b80-c772-11eb-914a-ad3775968062.gif)

Reply:
Actually, we could open a new GitHub issue for https://material-ui.com/components/data-grid/selection/#multiple-row-selection. Shouldn't <kbd>Shift</kbd> should allow range selection, as it does on Excel or on a Tree View?

Reply:
There's https://github.com/mui-org/material-ui-x/issues/208 but given the benchmark examples it seems more of a cell range selection. But I totally agree to allow the selection of a range of rows with <kbd>Shift</kbd>.

Reply:
Ok, I'm adding a screenshot to #208 to avoid confusion :). It's different